code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -80,8 +80,8 @@
 	. = ..()
 	var/slow = 0
 	if (bodytemperature < 283.222)
-		slow += (283.222 - bodytemperature) / 10 * 1.75
-	add_movespeed_modifier(MOVESPEED_ID_MONKEY_TEMPERATURE_SPEEDMOD, TRUE, 100, override = TRUE, multiplicative_slowdown = amount)
+		slow += ((283.222 - bodytemperature) / 10) * 1.75

```

Comment:
please excuse my dear aunt sally

this was causing them to literally freeze in place if they even got slightly chilly.

Reply:
Could definitely stand to be fine-tuned further, but this works for now.